src/components/menus/SmartAutoCorrect.ts

```
@@ -0,0 +1,804 @@
+        toggleAI?.addEventListener('click', (e) => {
+            e.stopPropagation();
+            this.toggleOption('useAI');
+            
+            // When enabling AI, check if OpenRouter needs validation
+            if (this.editor) {
+                const extensions = this.editor.extensionManager.extensions;
+                const smartAutoCorrectExt = extensions.find((ext: AnyExtension): ext is SmartAutoCorrectExtension => 
+                    ext.name === 'smartAutoCorrect' && 'options' in ext
+                );
+                
+                if (smartAutoCorrectExt && smartAutoCorrectExt.options.useAI) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)